### DISCUSSION TITLE
提问请一定要带上 CodeSandbox

### DISCUSSION THREAD

**Discussion by liuweiGL** [alibaba/formily#1469] [Ideas]
提问请一定要带上 CodeSandbox Demo，不然小伙伴们没法帮助你。

## antd 模板

1. Markup Schema：https://codesandbox.io/s/pxqz2
2. JSON Schema：https://codesandbox.io/s/jcpnx
3. 纯 JSX：https://codesandbox.io/s/7ql5g

## next 模板

1. Markup Schema：https://codesandbox.io/s/2s4br
2. JSON Schema：https://codesandbox.io/s/s9fb1
3. 纯 JSX：https://codesandbox.io/s/mq9n3

**Comment by workkk98** [comment]
[![CodeSandbox](https://img.shields.io/badge/CodeSandbox-000000?style=flat&logo=codesandbox&logoColor=white)](https://codesandbox.io/p/devbox/gracious-lehmann-4y5zsw-forked-5zjpsc?embed=1&file=%2Fsrc%2Fform.vue&layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clt40m5lj00063b6quhfxdco1%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clt40m5lj00023b6qo6768ejt%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clt40m5lj00043b6qvz4bdh45%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clt40m5lj00053b6qzeb5dm50%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clt40m5lj00023b6qo6768ejt%2522%253A%257B%2522id%2522%253A%2522clt40m5lj00023b6qo6768ejt%2522%252C%2522activeTabId%2522%253A%2522clt418lfs01db3b6qbakx7qgm%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clt40m5li00013b6qfdgooun8%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Fpackage.json%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%252C%257B%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Fwebpack.config.js%2522%252C%2522id%2522%253A%2522clt418lfs01db3b6qbakx7qgm%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%252C%257B%2522id%2522%253A%2522clt46qng600023b6nu9e6p2ed%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A32%252C%2522startColumn%2522%253A31%252C%2522endLineNumber%2522%253A32%252C%2522endColumn%2522%253A31%257D%255D%252C%2522filepath%2522%253A%2522%252Fsrc%252Fform.vue%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%257D%252C%2522clt40m5lj00053b6qzeb5dm50%2522%253A%257B%2522id%2522%253A%2522clt40m5lj00053b6qzeb5dm50%2522%252C%2522activeTabId%2522%253A%2522cltv5jjp500zu3b6qd14a428v%2522%252C%2522tabs%2522%253A%255B%257B%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A8080%252C%2522id%2522%253A%2522cltv5jjp500zu3b6qd14a428v%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522path%2522%253A%2522%252F%2522%257D%255D%257D%252C%2522clt40m5lj00043b6qvz4bdh45%2522%253A%257B%2522id%2522%253A%2522clt40m5lj00043b6qvz4bdh45%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clt40m5lj00033b6qc9i1cck3%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TERMINAL%2522%252C%2522shellId%2522%253A%2522cltv5bhsr0013dighemtva8at%2522%257D%255D%252C%2522activeTabId%2522%253A%2522clt40m5lj00033b6qc9i1cck3%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)
复现issue的过程中，创建了关于vue3、webpack、formily生态的模板。有需要可自取
